packages/notifications/package.json

```
@@ -0,0 +1,49 @@
+{
+  "name": "@terra-ui-packages/notifications",
+  "version": "0.0.1",
+  "scripts": {
+    "build": "vite build --emptyOutDir",
+    "dev": "vite build --mode=development --watch",
+    "test": "jest"
+  },
+  "type": "module",
+  "module": "./lib/es/index.js",
+  "main": "./lib/cjs/index.cjs",
+  "types": "./lib/types/index.d.ts",
+  "exports": {
+    ".": {
+      "import": "./lib/es/index.js",
+      "require": "./lib/cjs/index.cjs",
+      "types": "./lib/types/index.d.ts"
+    }
+  },
+  "files": [
+    "lib/cjs/**",
+    "lib/es/**",
+    "lib/types/**/*.d.ts"
+  ],
+  "dependencies": {
+    "@terra-ui-packages/components": "*",
```

```suggestion
    "@terra-ui-packages/components": "0.0.6",
```

Since this package is going to be published, it should specify a compatible version of components. Otherwise, components could update out from under it.

---

though taking in patch version ticks should be ok, right?

---

That's fine, but it's also the same thing. Caret is "do not modify the left most non-zero number", which currently for components is the patch version.
https://github.com/npm/node-semver?tab=readme-ov-file#caret-ranges-123-025-004

---

on a relate note, we should probably stop using 0.x.y versions to make semantic versioning fully "active" across our packages..

---

Yeah, we've been kicking the can down the road until there was a need to start thinking about it. With packages starting to depend on each other, I agree that time has arrived.